include/nebula/client/SessionPool.h

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2022 vesoft inc. All rights reserved.
+//
+// This source code is licensed under Apache 2.0 License.
+
+#pragma once
+
+#include <common/datatypes/DataSet.h>
+
+#include <cassert>
+#include <cstddef>
+#include <mutex>
+
+#include "nebula/client/Config.h"
+#include "nebula/client/ConnectionPool.h"
+#include "nebula/client/Session.h"
+
+namespace nebula {
+
+struct SessionPoolConfig {
+  std::string username_;
```

Comment:
Add checks to validate the config, such as the username/password should not be empty

Reply:
ok